src/App.js

```diff
@@ -39,15 +39,14 @@ export default function App() {
         <StylesProvider injectFirst>
           <CssBaseline />
           <BrowserRouter>
-            <HandleRedirectTo />
```

Comment:
This is a component I created before to handle ?redirectTo=... query params for the sake of making https://covidactnow.org/share/xyz/?redirectTo=/us/wa/compare/xyz work (since we need to point at /share/xyz/ to get the right meta tags).

I had to move it lower in the page to work around an issue where react-router-dom sometimes ignored it. I think there was a conflict between whether the redirect or one of our routes took effect first or something. 🤷